README.md

```diff
@@ -71,8 +72,9 @@ In addition to the provided unit tests, we provided a demo game application that
 You can start the demo game application with the following command:
 
 ```bash
-$ yarn run demo-game
+$ npm run demo-game
 ```
+(Note: This command **must** include `run` to in order to work since it is a customize script.)
```

Comment:
Same question as above.
```suggestion
(Note: This command **must** include `run` to in order to work since it is a custom script.)
```
Do we need this note? We don't talk much (at all?) about which of the scripts are well-known or not (custom), so I don't think there's much risk of a student trying to run `npm coverage` alone. And if they did, it just wouldn't work, and they'd run the correct command, which is what we gave them.

Reply:
True, true! I can remove these!